### ISSUE TITLE
pressing 'i' in telescope extension

### ISSUE THREAD

**Issue by jonestristand** [axkirillov/hbac.nvim#24]
Odd issue, can't quite track it down, but I get the following error if I open the telescope extension a second time and press i anywhere in the buffer name I am searching for... Has anyone seen similar?

```
E5108: Error executing lua: ...hare/nvim/lazy/telescope.nvim/lua/telescope/mappings.lua:269: attempt to call upvalue 'key_func' (a table value)
stack traceback:
        ...hare/nvim/lazy/telescope.nvim/lua/telescope/mappings.lua:269: in function <...hare/nvim/lazy/telescope.nvim/lua/telescope/mappings.lua:268>
```

**Comment by al-ce** [comment]
It's a bug because the `n` and `i` mode mappings tables are being copied recursively on subsequent calls. My fault! Would you please try the fix in this PR? #25

**Comment by jonestristand** [comment]
Yep, that fixes it!
